### PR TITLE
Issue #109 is resolved

### PR DESCRIPTION
**Pull request by PrathamDogra**
Issue #109 due to which the characters with lower serif(e.g. g ,y etc) get cut off from the bottom in the input fields of user.details is being resolved by adding a new class in <mat-form-field>  named input in details.component.html file and inside details.component.scss file changed the font size to 13px in class input which is by default 14px. Now the bottom of lower serif characters doesn't get clipped.
untitled

### CODE REVIEW

File: src/app/profile/user/details/details.component.scss [hunk 1]
```diff
@@ -229,3 +232,31 @@
...
+
+// .select {
+//   div.mat-input-infix {
+//     padding: .3em 0;
+//   }
+//   div.mat-select-value{
+//     height: 17px;
+//     vertical-align: top;
+//   }
+// }
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.scss]: Please remove these comments.

File: src/app/profile/user/details/details.component.scss [hunk 2]
```diff
@@ -153,6 +153,9 @@
       font-size: 1.8rem;
       margin-bottom: 1rem;
     }
+    &--body {
+      font-size: 13px;
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.scss]: Why is this required?

File: src/app/profile/user/details/details.component.scss (hunk 1 above)
Comment (PrathamDogra) [reply src/app/profile/user/details/details.component.scss]: Sure.

File: src/app/profile/user/details/details.component.scss (hunk 2 above)
Comment (PrathamDogra) [reply src/app/profile/user/details/details.component.scss]: Added it so that the font-size of characters during input remains same (13px) after the user saves the changes.

File: src/app/profile/user/details/details.component.scss [hunk 3]
```diff
@@ -153,6 +153,7 @@
       font-size: 1.8rem;
       margin-bottom: 1rem;
     }
+
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.scss]: Remove this extra line and you're good to go!

Comment (PrathamDogra) [reply src/app/profile/user/details/details.component.scss]: Removed this line.
